zmon-controller-app/src/main/resources/templates/index.html

```diff
@@ -120,6 +120,25 @@
         ineum('traceId', '<%= request.getAttribute("X-INSTANA-T") %>' );
     </script>
 
+    <!-- Lightstep EUM -->
+    <script src="/lib/opentracing/opentracing-browser.min.js" th:if="${lightstepEnabled}"></script>
+    <script th:inline="javascript" th:if="${lightstepEnabled}">
+      var lightstepConfig = /*[[${lightstepConfig}]]*/ { accessToken: "", componentName: "" };
+      (function(i,s,o,g,r,a,m){a=s.createElement(o),m=s.getElementsByTagName(o)[0];
+      a.async=1;a.src=g;a.dataset.init_global_tracer=true;
+      a.dataset.access_token=lightstepConfig.accessToken;
+      a.dataset.component_name=lightstepConfig.componentName;
+      a.dataset.xhr_instrumentation=true;
+      a.dataset.verbosity=3;
+      m.parentNode.insertBefore(a,m);
+      })(window,document,'script','/lib/lightstep/lightstep-tracer.min.js');
+    </script>
+    <!-- <script src="/lib/lightstep/lightstep-tracer.min.js" th:if="${lightstepEnabled}"
```

Comment:
can we delete this commented code?

Reply:
yeap already got rid of it in the latest commit :)